src/NuGet.Protocol.Catalog/Serialization/CatalogJsonSerialization.cs

```diff
@@ -9,7 +9,7 @@ internal static class CatalogJsonSerialization
     {
         public static JsonSerializer Serializer => JsonSerializer.Create(Settings);
 
-        public static JsonSerializerSettings Settings => new JsonSerializerSettings
+        private static JsonSerializerSettings Settings => new JsonSerializerSettings
```

Comment:
This is not building on my machine.. failing now on `FileCursor`. Is that file supposed to be deleted then?

Reply:
Not sure what you mean. CI seems to work fine.
Can you share the error message plz

Reply:
It's because the `Settings` is used inside `FileCursor`  which is marked as removed inside the `csproj` file. See the comment above on `NuGet.Protocol.Catalog.csproj`

Reply:
Yes I deleted because it wasn't used. It came from copying over the sample NuGet code